<launch>
  <arg name="range" default="10.0"/>
  <arg name="origin_x" default="0.0"/>
  <arg name="origin_y" default="0.0"/>

  <!-- main simulator node -->
  <node name="pedsim_people_sensor" pkg="pedsim_sensors" type="pedsim_people_sensor" output="screen">
    <param name="pose_initial_x" value="$(arg origin_x)"/>
    <param name="pose_initial_y" value="$(arg origin_y)"/>
    <param name="fov_range" value="$(arg range)" type="double"/>
    <param name="rate" value="25.0" type="double"/>
  </node>

</launch>
